packages/flutter/test/material/dropdown_menu_test.dart

```diff
@@ -2201,6 +2222,7 @@ void main() {
     await tester.enterText(find.byType(TextField), 'read');
     await tester.pump();
     checkExpectedHighlight(searchResult: 'Unread', otherItems: <String>['All', 'Read']); // Because "Unread" contains "read".
+    await tester.enterText(find.byType(TextField), '');
```

Comment:
Why do we add this line here?

Reply:
If we don't do this, for some weird reason, in newer drop down we still see `read` as input text first

Reply:
I haven't look, but having to update an existing test for a weird reason is usually related to a bug in the logic change introduced by the PR.

Reply:
I think the issue was in a previous version. The current version doesn't change custom search behaviour, and the test would still pass if I delete this line.